go-eicio/writer.go

```diff
@@ -3,12 +3,25 @@ package eicio
 import (
 	"encoding/binary"
 	"io"
+	"os"
 )
 
 type Writer struct {
 	byteWriter io.Writer
 }
 
+func Create(filename string) (*Writer, error) {
+	file, err := os.Create(filename)
+	if err != nil {
+		return nil, err
+	}
+	return NewWriter(file), nil
+}
+
+func (wrt *Writer) Close() {
+	wrt.byteWriter.(*os.File).Close()
```

Comment:
I would type-assert using the `io.Closer` interface instead:

```go
wrt.byteWriter.(io.Closer).Close()
```

also, for an `io.WriteCloser`, it's a really good idea to check that `Close()` didn't return any error (as `Close()` is where buffers are flushed to disk: you probably want to know if "saveMyData" didn't fail :P)